app/controllers/api/v3/twilio_sms_delivery_controller.rb

```diff
@@ -16,6 +16,10 @@ def update_params
     details
   end
 
+  def message_session_id
+    params["MessageSid"] || params["SmsSid"]
+  end
+
   def delivery_status
     params["SmsStatus"] || params["MessageStatus"] || TwilioSmsDeliveryDetail.results[:unknown]
```

Comment:
Should we invert this clause to

```suggestion
    params["MessageStatus"] || params["SmsStatus"] || TwilioSmsDeliveryDetail.results[:unknown]
```

as well?